packages/sw-cache-expiration/README.md

```diff
@@ -17,79 +17,92 @@ Browse sample source code in the [demo directory](https://github.com/GoogleChrom
 
 ### sw-cache-expiration
 
-[packages/sw-cache-expiration/src/index.js:21-21](https://github.com/GoogleChrome/sw-helpers/blob/a6f471056b52998c278dcb22dfc96169ea99932a/packages/sw-cache-expiration/src/index.js#L21-L21 "Source code on GitHub")
+[packages/sw-cache-expiration/src/index.js:21-21](https://github.com/GoogleChrome/sw-helpers/blob/43dea0fe7c5c2dca3500b988667de8b9ead46a3a/packages/sw-cache-expiration/src/index.js#L21-L21 "Source code on GitHub")
 
 sw-cache-expiration Module
 
 ### Behavior
 
-[packages/sw-cache-expiration/src/lib/behavior.js:25-156](https://github.com/GoogleChrome/sw-helpers/blob/a6f471056b52998c278dcb22dfc96169ea99932a/packages/sw-cache-expiration/src/lib/behavior.js#L25-L156 "Source code on GitHub")
+[packages/sw-cache-expiration/src/lib/behavior.js:48-263](https://github.com/GoogleChrome/sw-helpers/blob/43dea0fe7c5c2dca3500b988667de8b9ead46a3a/packages/sw-cache-expiration/src/lib/behavior.js#L48-L263 "Source code on GitHub")
 
-TODO Behavior Description
+**Examples**
 
-#### cacheName
+```javascript
+// Used as an automatically invoked as "behavior" by a RequestWrapper:
```

Comment:
That's unfortunately not easy to control; it relies on the templates used by the [`documentation`](https://www.npmjs.com/package/documentation) package.

Longer term we may not use `documentation`, and perhaps what we move to will have a better story in terms of controlling the output.